homeassistant/components/teslemetry/entity.py

```diff
@@ -192,3 +192,10 @@ def _value(self) -> int:
             .get(self.din, {})
             .get(self.key)
         )
+
+    @property
+    def has(self) -> bool:
+        """Return True if a specific value is in wall connector coordinator data."""
+        return self.key in self.coordinator.data.get("wall_connectors", {}).get(
+            self.din, {}
+        )
```

Comment:
It seems that this function is doing the same as accessing `_value() is not None`. Could that be put inside the new function or used instead of the new function?

Reply:
`_value` can be literally `None` in addition to not existing, which is why this requried a seperate function.

Reply:
Ok! 👍